### DISCUSSION TITLE
Lipupini

### DISCUSSION THREAD

**Discussion by lipupini** [vanjs-org/van#131] [Show and tell]
https://github.com/instalution/lipupini

A content aggregator / portfolio site with demo @ https://lipupini-demo.dup.bz/@example

The default frontend uses VanJS :) JS components are in the [Lukinview plugin](https://github.com/instalution/lipupini/tree/master/plugin/Lukinview/webroot/js/components/Grid)

Also listed in [LICENSE.md](https://github.com/instalution/lipupini/blob/master/LICENSE.md) and [README.md](https://github.com/instalution/lipupini#acknowledgements)

**Comment by Tao-VanJS** [comment]
👍👍👍 Nice work! Thanks for sharing!
